source/_integrations/dynalite.markdown

```diff
@@ -164,6 +176,76 @@ preset:
           required: false
           type: float
           default: 2.0
+template:
+  description: Set the default parameters for the templates.
+  required: false
+  type: map
+  keys:
+    room:
+      description: This is used to define a room that has a preset to turn on all the channels in the area and a preset to turn off.
+      required: false
+      type: map
+      keys:
+        room_on:
+          description: Preset to turn area on.
+          required: false
+          type: integer
+          default: 1
+        room_off:
+          description: Preset to turn area off.
+          required: false
+          type: integer
+          default: 4
+    trigger:
+      description: This is used to define a preset that acts as a trigger, such as a door buzzer. It only supports turning on.
+      required: false
+      type: map
+      keys:
+        trigger:
+          description: Preset to trigger the action.
+          required: false
+          type: integer
+          default: 1
+    timecover:
+      description: "This is used to define a cover that has 3 presets: open, close, and stop. Potentially can also use a channel that some systems (e.g. Control4) use to also send commands to open and close the cover. It uses the duration it takes to open or close to determine position. In addition, many times, these covers include tilt by opening or closing for a short time, so this can be defined as well."
```

Comment:
```suggestion
      description: "This is used to define a cover that has 3 presets: `open`, `close`, and `stop`. Potentially can also use a channel that some systems (e.g., Control4) use to also send commands to open and close the cover. It uses the duration it takes to open or close to determine position. In addition, many times, these covers include tilt by opening or closing for a short time, so this can be defined as well."
```